File: Project Templates/Custom/Universal App (Basic).xctemplate/Source/Main/___VARIABLE_classPrefix___MainViewController.m
//
//  ___FILENAME___
//  ___PROJECTNAME___
//
//  Created by ___FULLUSERNAME___ on ___DATE___.
//  ___COPYRIGHT___
//

@import UIKit;

#import "___VARIABLE_classPrefix___MainViewController.h"

@implementation ___VARIABLE_classPrefix___MainViewController

-(instancetype)init {

	if (self = [super init]) {

		self.title = @"___PROJECTNAME___";
	}

	return self;
}

#if TARGET_OS_XR
-(UIContainerBackgroundStyle)preferredContainerBackgroundStyle {
	return UIContainerBackgroundStyleGlass;
}
#endif

@end

File: Project Templates/Custom/Universal App (Basic).xctemplate/Source/Main/___VARIABLE_classPrefix___MainViewController.swift
//
//  ___FILENAME___
//  ___PROJECTNAME___
//
//  Created by ___FULLUSERNAME___ on ___DATE___.
//  ___COPYRIGHT___
//

import UIKit

final class ___VARIABLE_classPrefix___MainViewController: UIViewController {

    init() {
        super.init(nibName: nil, bundle: nil)
        title = "___PROJECTNAME___"
    }

    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
	
	#if os(xrOS)
	override var preferredContainerBackgroundStyle: UIContainerBackgroundStyle {
		return .glass
	}
	#endif
}
